homeassistant/components/subaru/strings.json

```
@@ -0,0 +1,31 @@
+{
+  "config": {
+    "error": {
+      "connection_error": "Error connecting; check network and retry",
+      "identifier_exists": "Email already registered",
+      "invalid_credentials": "Invalid credentials",
+      "unknown_error": "Unknown error, please report log info"
```

```suggestion
      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
      "unknown": "[%key:common::config_flow::error::unknown%]"
      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
```

And, as mentioned previously, the `already_configured` case might be an abort.

---

Also, where is `unknown_error` used?